src/libraries/System.Data.Common/src/System/Data/SQLTypes/SqlXml.cs

```diff
@@ -518,10 +518,9 @@ private void ThrowIfStreamCannotWrite(string method)
                 throw new NotSupportedException(SQLResource.InvalidOpStreamNonWritable(method));
         }
 
-        private void ThrowIfStreamClosed(string method)
+        private void ThrowIfStreamClosed()
         {
-            if (IsStreamClosed())
-                throw new ObjectDisposedException(SQLResource.InvalidOpStreamClosed(method));
+            ObjectDisposedException.ThrowIf(IsStreamClosed(), this);
```

Comment:
NIT: looks the `SQLResource.InvalidOpStreamClosed(method)` was only used here (could be removed)